tests/resources/test_issue.py

```diff
@@ -26,6 +26,28 @@ def test_issue(self):
         self.assertEqual(issue.key, self.issue_1)
         self.assertEqual(issue.fields.summary, "issue 1 from %s" % self.project_b)
 
+    def test_issue_search_field(self):
+        issues = self.jira.search_issues("key=%s" % self.issue_1)
+        self.assertEqual(self.issue_1, issues[0].key)
+        self.assertIsInstance(issues, list)
+
+        issues = self.jira.search_issues("key=%s" % self.issue_1, json_result=True)
+        self.assertIsInstance(issues, dict)
+
+        issues = self.jira.search_issues(
```

Comment:
Could you also include a brief comment to explain what each block is testing

e.g.
```suggestion

        # Check that a field we do not request does not appear
        issues = self.jira.search_issues(
```

I think it may also be possible to do these as individual test functions
e.g. `issues[0]` is the same as `self.issue_1` (as we proved in the first assertion in this test)

That way we can specify in the function name itself what we are testing

`test_issue_search_finds_issue`
`test_issue_search_can_return_json`
`test_issue_search_only_includes_provided_fields`
`test_issue_search_default_behaviour_included_fields`